### PR TITLE
Added code for sort color

### PR DESCRIPTION
**Pull request by Anshika-Srivastava**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Added code for sort color.
[Issue 2653](https://github.com/TesseractCoding/NeoAlgo/issues/2653)
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/cp/Sort_Color.cpp [hunk 1]
```diff
@@ -0,0 +1,89 @@
+//Given an array nums with n objects colored red, white, or blue, 
+//sort them in-place so that objects of the same color are adjacent, with the colors in the order red, white, and blue.
+
+We will use the integers 0, 1, and 2 to represent the color red, white, and blue, respectively.
```
Comment (iamrajiv) [inline C-Plus-Plus/cp/Sort_Color.cpp:5]: @Anshika-Srivastava I think this should be a comment.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Anshika-Srivastava Add space and time complexity. Also, add the problem link. A lot of indentation is there in the code please format your code using some code formatter.

**Comment by Anshika-Srivastava** [comment]
**@iamrajiv I have already mentioned the issue link in the first comment. Please review it once.**

**Comment by iamrajiv** [comment]
@Anshika-Srivastava I am talking of link of the problem means website or platform link so that people can redirect to question easily.

**Comment by Anshika-Srivastava** [comment]
**@iamrajiv Added link to the question as well as examples. Please review it. Thank you!**

**Review by iamrajiv — APPROVED** [review]

**Comment by iamrajiv** [comment]
@Anshika-Srivastava Please resolve the merge conflict so that I can merge the PR.

**Comment by Anshika-Srivastava** [comment]
Can you tell me like whats the problem here @iamrajiv 
image

**Comment by iamrajiv** [comment]
@Anshika-Srivastava A merge conflict is an event that occurs when Git is unable to automatically resolve differences in code between two commits means someone edited the readme as you did but his/her PR is merged first so there is an update in the file but since your PR is still in the review so your readme file is outdated which causes merge conflict. You can resolve the conflicts by huts clicking `Resolve conflicts` and cleaning the `<<<<<<< issue2653` and `=======` and `>>>>>>> master` lines but for future work, I would just of using the command line to resolve the merge conflicts rather than directly.

**Comment by Anshika-Srivastava** [comment]
**Done! Thank you so much!!
Please Merge It!**

**Pull request merged**